ShovIP/app/src/main/java/com/example/shovip/DialPadFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.shovip
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import com.example.shovip.databinding.FragmentDialPadBinding
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class DialPadFragment : Fragment() {
+
+    private var _binding: FragmentDialPadBinding? = null
+
+    // This property is only valid between onCreateView and
+    // onDestroyView.
+    private val binding get() = _binding!!
```

Comment:
Is this getter required?

Also, `!!` operator may throw NPE.

I would suggest to make `binding` a non-null member (if that is possible), then this getter can be removed

Reply:
why there's `!!` operator?

Reply:
See https://github.com/shovian/ShovIP/pull/1#discussion_r682256391